channels/views_test.py

```diff
@@ -698,6 +698,15 @@ def test_list_moderators(client, use_betamax, praw_settings):
     assert resp.json() == [{'moderator_name': 'fooadmin'}]
 
 
+def test_list_subscribers_not_allowed(client):
+    """
+    Get method not allowed on the list of subscribers
+    """
+    client.force_login(UserFactory.create(username='fooadmin'))
```

Comment:
You should use `jwt_header` fixture instead (see my comment on the next test).  That one is for a nonstaff user.